### ISSUE TITLE
Land use intensity calculation not consistent with BRAT

### ISSUE THREAD

**Issue by mhallerud** [Riverscapes/RCAT#14]
@wally-mac, Jenna Walsh (from the USFS PIBO team) and I have been investigating RCAT outputs for the PIBO project and have noticed a few inconsistencies between RCA. RVD, and BRAT. For land use intensity, BRAT's land use intensity is on a scale from 0-1 with 0 being the most natural and 1 being the most developed, while RCAT's land use intensity is on a 0-1 scale with 0 being the most developed and 1 being the most natural. RCAT's scale seems counter-intuitive to us, plus @philipbaileynar is still developing BRAT 3 and we don't want to confuse end users who have recent outputs of BRAT. So we're going to change the RCAT land use intensity to match BRAT's LUI scale so that they're consistent.

**Comment by mhallerud** [comment]
@wally-mac An update on this- the RCAT system of land use intensity classification seems completely inconsistent and doesn't make a ton of sense to me. The classifications are as follows, I bolded the ones I found particularly confusing:

1 = **barren**, conifer, conifer-hardwood, **exotic tree-shrub**, grassland, hardwood, open water, riparian, shrubland, snow-ice, sparsely vegetated
0.66 = **developed**, **exotic herbaceous**, agricultural (fallow/idle, pasture/hayland, wheat, **aquaculture**, **cultivated crops and irrigated agriculture** 
0.33 = **quarries-strip mines-gravel pits**, agricultural (orchard, vineyard, berries, row crops) 
0 =  developed-high intensity, developed-low intensity, developed-roads

Managed tree plantations, which are 0.66 ("high intensity agriculture") are not included in the RCAT classification at all, so would get a value of 0 (developed) by default. I'm wondering if it would make sense to just pull the LANDFIRE_LUCode from the BRAT supporting tools and add it into RCAT so that they're 100% consistent?
